articles/quickstart/backend/php/01-authorization.md

```diff
@@ -248,4 +254,4 @@ $router->get('/api/private-scoped', function() use ($app){
 });
 ```
 
-The route `/api/private-scoped` will be accessible only if has a valid Access Token with the scope `read:messages`.
\ No newline at end of file
+The route `/api/private-scoped` will be accessible only if has a valid Access Token with the scope `read:messages`.
```

Comment:
This line doesn't appear to actually be changed, but maybe good to fix the grammar while we're here. Should it be "The route... will be accessible only if the request contains a valid Access Token with the scope..."?

Reply:
That does sound better, good call 👍